Sources/Hummingbird/Application.swift

```diff
@@ -178,4 +124,44 @@ public final class HBApplication: HBExtensible {
             try self.eventLoopGroup.syncShutdownGracefully()
         }
     }
+
+    public func addChannelHandler(_ handler: @autoclosure @escaping @Sendable () -> any RemovableChannelHandler) {
+        self.additionalChannelHandlers.append(handler)
+    }
+}
+
+/// Conform to `Service` from `ServiceLifecycle`.
+/// TODO: Temporarily I have added unchecked Sendable conformance to the class as Sendable
```

Comment:
Why is this unchecked Sendable necessary>

Reply:
As the comment says the protocol `Service` requires `HBApplication` to be `Sendable`. There is no way I can set it to `Sendable` now without major structural change. I will revisit this later probably when I split the applicatino into a builder type and a running type. The running type will be `Sendable` and can conform to `Service` without the `@unchecked Sendable`